maple-story.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>maple-story</title>
+    <style>
+      .title {
+        color: orange;
+      }
+    </style>
+  </head>
+  <body>
+    <h1 class="title">maple-story</h1>
+    <p>
+      메이플 스토리는 넥슨이라는 회사에서 2003년 4월 29일부터 상용 서비스를
+      시작했다.
+      <br />
+      SD 생김새의 캐릭터가 모험을 떠나면서 성장하는 2D 횡스크롤 게임이다.
+      <br />
+      메이플이란 게임의 직업에는 여러가지가 있다
```

Comment:
말을 하다가 말았습니다 애초에 지우거나 설명을 이어서하는게 나을거 같아요.
```suggestion
      메이플이란 게임의 직업에는 여러가지가 있다
      도적,궁수 등등
```